### PR TITLE
notmuch: Allow to use untransformed tag for color

### PR DESCRIPTION
**Pull request by sileht**
Hi,

This change allow to use untransformed tag for index-tag color.

Cheers,

### CODE REVIEW

**Comment by flatcap** [comment]
Hmm... there are too many `(`s

**Comment by flatcap** [comment]
@neomutt/reviewers Please Review

File: menu.c [hunk 1]
```diff
@@ -52,6 +52,9 @@ get_color (int index, unsigned char *s)
     case MT_COLOR_INDEX_TAG:
       for (color = ColorIndexTagList; color; color = color->next)
       {
+        if (strncmp((const char *)(s+1), color->pattern,
+                    strlen(color->pattern)) == 0)
+          return color->pair;
         const char * transform = hash_find(TagTransforms, color->pattern);
```
Comment (gahr) [inline menu.c]: (perhaps my previous comment went missing)

Are you sure that `strncmp` is correct? Should "red" match "redish"?

Comment (flatcap) [inline menu.c]: That's what I thought, but the pattern is being compared to the line which is "TAG" followed by a binary colour code.

Comment (gahr) [inline menu.c]: Alright, thanks for the explanation!

**Pull request merged**